docs/user/.gitignore

```diff
@@ -0,0 +1 @@
+search-syntax.md
```

Comment:
Would `*.md` be better?
So when there is a new page, there is no need to update this file

Reply:
See my comment above https://github.com/searxng/searxng/pull/930#discussion_r820058125